tests/infrastructure/test_process.py

```diff
@@ -59,6 +59,14 @@ def test_process_runs_command(tmp_path):
     assert (tmp_path / "canary").exists()
 
 
+def test_process_runs_command_in_working_dir(tmp_path, capsys):
```

Comment:
You can use `tmpdir` instead of `tmp_path` if you want, it's already a string instead of a `Path` object.

Reply:
I did use tmpdir ... it's a `LocalPath` and not a string, but I decided to add a test for `Path` and `str` so I believe we're covered.